apps/meteor/server/models/raw/LivechatRooms.ts

```diff
@@ -2543,6 +2544,54 @@ export class LivechatRoomsRaw extends BaseRaw<IOmnichannelRoom> implements ILive
 			.toArray();
 	}
 
+	async countLivechatRoomsByPriority(): Promise<IStats['totalLivechatRoomsWithPriority']> {
+		const pipeline = [
+			{
+				$match: {
+					t: 'l',
+				},
+			},
+			{
+				$group: {
+					_id: '$priorityWeight',
+					count: { $sum: 1 },
+				},
+			},
+			{
+				$project: {
+					_id: 0,
+					lowest: { $cond: [{ $eq: ['$_id', LivechatPriorityWeight.LOWEST] }, '$count', 0] },
+					low: { $cond: [{ $eq: ['$_id', LivechatPriorityWeight.LOW] }, '$count', 0] },
+					medium: { $cond: [{ $eq: ['$_id', LivechatPriorityWeight.MEDIUM] }, '$count', 0] },
+					high: { $cond: [{ $eq: ['$_id', LivechatPriorityWeight.HIGH] }, '$count', 0] },
+					highest: { $cond: [{ $eq: ['$_id', LivechatPriorityWeight.HIGHEST] }, '$count', 0] },
+					notSpecified: { $cond: [{ $eq: ['$_id', LivechatPriorityWeight.NOT_SPECIFIED] }, '$count', 0] },
+				},
+			},
+			{
+				$group: {
+					_id: null,
+					lowest: { $sum: '$lowest' },
+					low: { $sum: '$low' },
+					medium: { $sum: '$medium' },
+					high: { $sum: '$high' },
+					highest: { $sum: '$highest' },
+					notSpecified: { $sum: '$notSpecified' },
+				},
+			},
+		];
+
+		const result = await this.col.aggregate(pipeline).toArray();
+		const stats = result[0];
+		delete stats._id;
```

Comment:
You don't need to manually delete `_id`, you can `project { _id: 0 }` on the agg